### PR TITLE
Adjust custom events

### PR DESCRIPTION
**Pull request by creocoder**

### CODE REVIEW

File: spec/definitions/CustomEvents/CustomEvent.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ properties:
       - before
       - after
   scheduleInstruction:
-    $ref: "#/definitions/ScheduleMethod"
+    $ref: "#/definitions/CustomEventScheduleInstruction"
```
Comment (slavcodev) [inline spec/definitions/CustomEvents/CustomEvent.yaml]: What is the reason of this change? Why the exiting one can't be used?

Comment (creocoder) [inline spec/definitions/CustomEvents/CustomEvent.yaml]: @slavcodev Not all values are supported.
